libindy/src/api/wallet.rs

```diff
@@ -297,6 +297,53 @@ pub extern fn indy_list_wallets(command_handle: i32,
     res
 }
 
+/// Exports opened wallet
+///
+/// #Params:
+/// handle: wallet handle returned by indy_open_wallet
+/// export_path: file path where wallet should be exported to
+/// export_key: key from which export key will be derived
+///
+/// #Returns
+/// Error code
+///
+/// #Errors
+/// Common*
+/// Wallet*
+#[no_mangle]
+pub extern fn indy_export_wallet(command_handle: i32,
+                                 handle: i32,
+                                 export_path: *const c_char,
```

Comment:
I suggest to replace 

> export_path: *const c_char,
> export_key: *const c_char,

with export_config_json to allow extend export configuration in a backward compatible way